modAionImpl/src/org/aion/zero/impl/sync/SyncStats.java

```diff
@@ -230,6 +230,74 @@ public void updatePeerTotalBlocks(String _nodeId, int _totalBlocks) {
         }
     }
 
+    /**
+     * Updates the total number of blocks imported from each seed peer
+     *
+     * @param _nodeId peer node display Id
+     * @param _importedBlocks total number of blocks imported
+     */
+    public void updatePeerImportedBlocks(String _nodeId, int _importedBlocks) {
+        seedsLock.lock();
+        try {
+            long blocks = (long) _importedBlocks;
+            if (importedByPeer.putIfAbsent(_nodeId, blocks) != null) {
+                importedByPeer.computeIfPresent(_nodeId, (key, value) -> value + blocks);
+            }
+        } finally {
+            seedsLock.unlock();
+        }
+    }
+
+    /**
+     * Obtains the total number of blocks imported from the given seed peer
+     *
+     * @return number of total imported blocks by peer
+     */
+    long getImportedBlocksByPeer(String _nodeId) {
+        seedsLock.lock();
+        try {
+            return this.importedByPeer.get(_nodeId);
+        } catch (NullPointerException e) {
+            return 0;
+        } finally {
+            seedsLock.unlock();
+        }
+    }
+
+    /**
+     * Updates the total number of blocks stored from each seed peer
+     *
+     * @param _nodeId peer node display Id
+     * @param _storedBlocks total number of blocks stored
+     */
+    public void updatePeerStoredBlocks(String _nodeId, int _storedBlocks) {
+        seedsLock.lock();
```

Comment:
please update this method to do nothing when `seedEnabled` is `false` (similar to `updatePeerTotalBlocks`)